docs/reference-guides/cluster-configuration/rancher-server-configuration/k3s-cluster-configuration.md

```diff
@@ -98,6 +98,14 @@ Select the domain for the cluster. The default is `cluster.local`.
 
 Option to change the range of ports that can be used for [NodePort services](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport). The default is `30000-32767`.
 
+#### Truncate Hostnames
+
+Option to truncate hostnames, for NetBIOS compatibility. Although Kubernetes allows hostnames up to 63 characters in length, Windows systems have a hard limit of 15 characters. 
```

Comment:
> Windows systems have a hard limit of 15 characters
I'd say it's a bit too strong of a statement - it's not a true hard limit for all configurations. So recommending to something along the lines "setups including Windows nodes have better compatibility (with NetBIOS) when hosnames don't exceed 15 characters.

Also, we want to ensure it's called out this option is only for machine-provisioned (not Custom) clusters - as for the latter hostnames are set outside of Rancher when nodes are created.

Reply:
I just pushed [c46e6e4](https://github.com/rancher/rancher-docs/pull/664/commits/c46e6e4815d478a4a1b95d3d977cefef56aab2aa) to address this, as well as removing the line about how you can manually edit the limit